src/test/java/src/WordleDictionaryTest.java

```diff
@@ -0,0 +1,33 @@
+package src;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class WordleDictionaryTest {
+
+    @Test
+    void getList() throws FileNotFoundException {
+        //given
+        WordleDictionary diction = new WordleDictionary("C:\\Users\\User\\Downloads\\dictionary.txt");
```

Comment:
I told you on the call you couldn't do this and you needed to have dictionary.txt in your repo.